src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable, AreaCalculable {
```

Comment:
The class name 'Figure' is appropriate as it is a noun, but ensure that the interfaces 'Drawable' and 'AreaCalculable' do not contain verbs. If they do, consider renaming them to follow the convention, such as 'Drawable' to 'DrawableFigure' and 'AreaCalculable' to 'AreaCalculator' if they contain verbs.